crypto/fipsmodule/cpucap/cpu_aarch64_openbsd.c

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) 2022, Robert Nagy <robert@openbsd.org>
```

Comment:
is this file copied from OpenBSD?

Reply:
This file is copied from the set of patches which OpenBSD applies to Chromium in order to build Chromium on OpenBSD. The author of this particular patch happens to be an OpenBSD committer.